src/content/docs/vulnerability-management/changing-vulnerability-status.mdx

```diff
@@ -0,0 +1,293 @@
+---
+title: Changing vulnerability status
+metaDescription: Use Vulnerability Management to overcome blindspots and assign remediation to developers as a security team.
+freshnessValidatedDate: never
+---
+
+import vmIgnoreEntityVulnList from 'images/security_screenshot-ignore-entity-vulnlist.webp'
+
+import vmIgnoreEntityDetails from 'images/security_screenshot-ignore-entity-details.webp'
+
+import vmIgnoreEntityForm from 'images/security_screenshot-ignore-entity-form.webp'
+
+import vmIgnoreBulkVulnList from 'images/security_screenshot-ignore-bulk-vulnlist.webp'
+
+import vmIgnoreBulkDetails from 'images/security_screenshot-ignore-bulk-details.webp'
+
+import vmIgnoreBulkForm from 'images/security_screenshot-ignore-bulk-form.webp'
+
+import vmAffectedEntityVulnList from 'images/security_screenshot-affected-entity-vulnlist.webp'
+
+import vmAffectedEntityDetails from 'images/security_screenshot-affected-entity-details.webp'
+
+import vmAffectedEntityFiltered from 'images/security_screenshot-affected-entity-filtered.webp'
+
+import vmAffectedEntityForm from 'images/security_screenshot-affected-entity-form.webp'
+
+import vmAffectedBulkVulnList from 'images/security_screenshot-affected-bulk-vulnlist.webp'
+
+import vmAffectedBulkSelection from 'images/security_screenshot-affected-bulk-selection.webp'
+
+import vmAffectedBulkDetails from 'images/security_screenshot-affected-bulk-details.webp'
+
+import vmAffectedBulkForm from 'images/security_screenshot-affected-bulk-form.webp'
+
+Many organizations have an overwhelming number of vulnerabilities detected within their system. But, not all of these vulnerabilities are high-risk, have remediations, or are run on systems of high operational importance or external exposure. NRVM allows you to ignore vulnerabilities on a per-entity basis, this allows for dashboards to only contain vulnerabilities that match your risk threshold and ensure your teams focus on impactful remediations. 
+
+## Requirements 
+
+- Vulnerability Management requires 'write' permissions for the [vulnerabilities capability](/docs/accounts/accounts-billing/new-relic-one-user-management/user-permissions/#security) in order for a user to change the status of a vulnerability.
```

Comment:
Is there a version requirement for agents or for vulnerability management that customers need to be on?

Reply:
Does VM with this feature happen out of the box on install?

Reply:
There is a general doc for NRVM agent versions. (https://docs.newrelic.com/docs/vulnerability-management/integrations/intro/#apm-agents) 
This feature happens out of the box on install but requires users have the 'write' permission. Custom roles will not have this permission by default.